apps/wallet/src/ui/app/pages/home/transfer-coin/index.tsx

```diff
@@ -64,7 +64,7 @@ function TransferCoinPage() {
                 });
 
                 return signer.signAndExecuteTransaction({
```

Comment:
should we also rename this to `signAndExecuteTransactionBlock`?

Reply:
Yeah happening in this follow-up: https://github.com/MystenLabs/sui/pull/9946